Sokoliuk/HW6/HW6_task_01.py

```diff
@@ -0,0 +1,22 @@
+numbers = list(range(1, 11))
+ls_even, ls_odd, ls_other = [], [], []
+
+ls_even = [number for number in numbers if number % 2 == 0]
+print("Even numbers that divisible by 2: ", ls_even)
+
+ls_odd = [number for number in numbers if number % 3 == 0]
+print("Odd numbers, which are divisible by 3: ", ls_odd)
+
+ls_other = [number for number in numbers if number % 3 and number % 2 != 0]
+print("Numbers that are not divisible by 2 and 3: ", ls_other)
+
+# for number in numbers:
+#     if number % 2 == 0:
+#         ls_even.append(number)
+#     elif number % 3 == 0:
+#         ls_odd.append(number)
+#     else:
```

Comment:
look at this version of code
l1 = [ x for x in range(1, 11) if x % 2 == 0 ]
l2 = [ x for x in range(1, 11) if x % 2 == 1 and x % 3 == 0 ]
l3 = [ x for x in range(1, 11) if not x % 2 == 0 and not x % 3 == 0 ]
print(l1, 'is even multiple of 2')
print(l2, 'is an odd multiple of 3')
print(l3, 'not divisible by 2 and 3')